swarm/src/connection/pool.rs

```diff
@@ -783,13 +791,14 @@ where
                         )
                         .boxed(),
                     );
-
+                    let time_taken = Instant::now() - accepted_at;
```

Comment:
```suggestion
                    let established_in = accepted_at.elapsed();
```

Let's use [`elabsed`](https://doc.rust-lang.org/nightly/std/time/struct.Instant.html#method.elapsed) here.